README.md

```diff
@@ -310,6 +310,11 @@ added to the `CMAKE_PREFIX_PATH` when building SYCL-BLAS (see
 
 ## Setup
 
+**IMPORTANT NOTE:** The `TARGET` CMake variable is no more supported, it has
+been replaced by `TUNING_TARGET`, which accepts the same options.
+`TUNING_TARGET` does not effect compilation, but only the tuning configuration.
```

Comment:
I don't understand why we say that `TUNING_TARGET` doesn't affect compilation. It is still used to determine which file to include in level 2 or 3 [backend.hpp](https://github.com/codeplaysoftware/sycl-blas/blob/master/src/interface/blas3/backend/backend.hpp). I think you want to say that it is not affecting the sycl target anymore. I'm not sure it's worth explaining that, we could just say that `TARGET` is not supported.
Same comment for `ConfigureSYCLBLAS.cmake`.

Reply:
Good point!